src/lv_ex_widgets/lv_ex_gauge/lv_ex_gauge_2_sim.py

```diff
@@ -0,0 +1,30 @@
+from lv_colors import lv_colors
+
+with open('../../../assets/img_hand.bin','rb') as f:
```

Comment:
When accessing files like that, does it work with the online simulator?
I believe that you need to provide a URL for the simulator, otherwise the "Try It Now" button on the doc page would not work.
If you are not sure, we can fix that later after this is deployed on the help docs.

Reply:
Correct! I said this in my first comment on this thread. 
I have 2 versions of the programs using images:
ex_img_png.py
ex_img.py
The versions with png in their name do not run on the simulator (problem to convert png file) while the others do.
However, the image file names must still be converted to URLs. This however I can only test once the URLs exist in lv_examples

Reply:
> However, the image file names must still be converted to URLs. This however I can only test once the URLs exist in lv_examples

Ok, we can fix these paths to URLs after this PR is merged.